services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts

```diff
@@ -42,10 +45,12 @@ export default class LiveSyncerService {
     private readonly dagService: DagService,
     private readonly pbftService: PbftService,
     private readonly txService: TransactionService,
-    private readonly graphQLConnector: GraphQLConnectorService
+    private readonly graphQLConnector: GraphQLConnectorService,
+    private readonly historicalSyncerService: HistoricalSyncService
   ) {
     this.logger.log('Starting NodeSyncronizer');
     this.isWsConnected = false;
+    this.queueCache = new QueuePopulatorCache(this.pbftsQueue, 10);
```

Comment:
If we use this in both modules, maybe we can move it to a separate, `common`, module